org_chart.js

```diff
@@ -1,73 +1,56 @@
-let elon = {
-  name: "Elon Musk",
-  email: "elon.musk@musky.com",
-  position: "Boss",
+//CONSTRUCTOR AND PROTOTYPE////////////////
+function Hire(name, position, project, directory, username = name.replace(' ', '.').replace('-', '_').toLowerCase()) {
+    this.name = name;
+    this.email = username + '@musky.com';
+    this.position = position;
+    this.project = project;
+    this.skills = [];
```

Comment:
To address your comment: the `skills` property is an array because we may want to add more skills to a person.
That isn't explicitly shown in the example, but that is an array should hint that the intent is to allow more than one